extension/src/fileSystem/index.test.ts

```diff
@@ -408,3 +409,51 @@ describe('findOrCreateDvcYamlFile', () => {
     )
   })
 })
+
+describe('isPathInProject', () => {
+  it('should return true if the path is in the project', () => {
+    const path = join(dvcDemoPath, 'dvc.yaml')
+    const dvcRoot = dvcDemoPath
+    const subProjects: string[] = []
+    expect(isPathInProject(path, dvcRoot, subProjects)).toBe(true)
+  })
```

Comment:
Looks like we're missing a new line here:
```suggestion
  })
  
```

Reply:
I wonder if there is a prettier rule for this

Reply:
maybe this https://eslint.org/docs/latest/rules/padding-line-between-statements